packages/graphql/package.json

```diff
@@ -22,7 +22,7 @@
         "dist/**/*.js.map"
     ],
     "engines": {
-        "node": ">=16.0.0"
+        "node": ">=20.0.0"
```

Comment:
Shall we bump also the introspector/ogm packages?

Reply:
I have only adjusted what is here... 🫠 We don't currently have this in the `package.json` of the other packages! Presumably because the GraphQL Library is a dependency of them.